content/en/blog/2025/sail-operator-ga/index.md

```diff
@@ -0,0 +1,301 @@
+---
+title: "Sail operator 1.0.0 has been released: manage Istio with an operator"
```

Comment:
per https://istio.io/latest/blog/2024/introducing-sail-operator/ and https://github.com/istio-ecosystem/sail-operator, it's "Sail Operator" throughout

Reply:
(Please make sure Istio is capitalized where appropriate too, in case I missed any)